src/upgrade/upgrade_scripts/5.15.6/upgrade_bucket_policy.js

```diff
@@ -8,7 +8,11 @@ const _ = require('lodash');
 
 function _create_actions_map() {
     const actions_map = new Map();
-    for (const action of Object.values(OP_NAME_TO_ACTION)) {
+    // since we didn't have this API in version 5.14.0 and now the map includes array
```

Comment:
I am a little unsure here. Are we going to backport this feature to previous NooBaa versions? If not then do we need to do any kinds of delete here as those versions wont' have `get_object_attributes` right?

Reply:
I changed the code according to Guy's comment ([here](https://github.com/noobaa/noobaa-core/pull/8418#discussion_r1799471206)). I hope it is clearer.
@nimrod-becker as I understand there is no plan to backport this PR, right?
cc: @guymguym @tangledbytes @romayalon